javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,9 @@
+function getFullName(firstname, surname, useFormalName=false, maleOrFemale=true){
+    const sex = maleOrFemale ? "Lord" : "Majesty";
```

Comment:
I really like your structure here: nice and clean! 🤩 I would reconsider the variable name `maleOrFemale` as it is not clear what setting it to true or false means.

Reply:
yeah you are right
always have problem with naming
now it is isItMale